### DISCUSSION TITLE
Logout no longer possible with v3.15.0 resp. v3.15.1

### DISCUSSION THREAD

**Discussion by ribeaud** [encode/django-rest-framework#9347] [Potential Issue] [answered]
Using `/backend` path (**API** endpoint), it is no longer possible to logout. The popup menu window does no longer appear when trying to open the select box. Opening the developer console, I see following output:

<img width="694" alt="Screenshot_2024-03-18_at_13 57 40" src="https://github.com/encode/django-rest-framework/assets/13186156/c78cc4e6-bbdd-4f54-a71e-6100744cbc9e">

Am I the only one? Going back to _3.14.x_ solves the issue. _3.15.1_ does NOT solve it.

**Comment by realsuayip** [comment, marked as answer]
Some of the static files are renamed/updated (in this case, new file is `jquery-3.7.1.min.js`)

Try clearing your browser cache and it should be fine. If you are using browsable API in production, you'll need to run `collectstatic` too.

**Reply by ribeaud** [reply]
Thank you, that was useful. I was not aware, we are using our own template. I will need to adapt it.

**Reply by lovelydinosaur** [reply]
> I was not aware, we are using our own template. I will need to adapt it.

Ah that's interesting yep, I'm not sure we can really guard against that.
We *might* choose to mention JS/CSS changes in the release notes in the future to help folks using custom templates.

(I don't think the browser cache needs clearing... I can see [that the templates](https://github.com/encode/django-rest-framework/pull/9094/files) are linking to the updated version of jQuery, and can see the request being made and 404'ing. You do need to run collectstatic tho, yep.)
